lib/tizen_sdk.dart

```diff
@@ -300,9 +315,18 @@ class TizenSdk {
     Rootstrap rootstrap = getRootstrap(profile, apiVersion, type);
     if (!rootstrap.isValid && profile == 'tv-samsung') {
       _logger.printTrace('TV SDK could not be found.');
-      profile = 'iot-headed';
+      if (versionToDouble(apiVersion) >= 8.0) {
+        profile = 'tizen';
+      } else {
+        if (arch == 'x86') {
+          profile = 'mobile';
```

Comment:
The requirement for the mobile rootstrap should be specified in `tizen_doctor.dart` and the docs if it's going to be the new default.